client/components/library-items.vue

```diff
@@ -229,6 +251,9 @@ export default {
             };
             bus.$emit('initSpeedbump', callback, speedbumpOptions);
         },
+        clearSearch() {
+            this.searchText = "";
+        }
```

Comment:
this would not be necessary if using `<input type="search">`
```suggestion
```

Reply:
The only reason I chose to use type="text" is because the native "x" button that comes with type="search" doesn't look good with the dark color of the side-bar. 
![native_x](https://user-images.githubusercontent.com/48631336/85341329-80b66380-b4ad-11ea-9f63-f802495ae2ed.png)
And I had a hard time customizing it in any way. 

I was able to make the custom one look much cleaner:
![Annotation 2020-06-22 173050](https://user-images.githubusercontent.com/48631336/85341594-2b2e8680-b4ae-11ea-83d3-23295cce937a.png)


Still, I'm happy to take another stab at type="search" if you think that's the way to go.

Reply:
yeah thats weird, not sure why its showing up blue, when i inspect i dont see the correlating style. You could style it using the sudo selector `-webkit-search-cancel-button` but thats just webkit. Yeah i like your approach better